Menus/MainMenu/MainMenu.cs

```diff
@@ -121,6 +109,25 @@ void Start()
 			UpdateToolButtons();
 		}
 
+		public void SetVisible(bool visible, bool temporary = false)
+		{
+			if (m_Visible != visible)
+			{
+				m_Visible = visible;
+				if (m_MainMenuUI)
+				{
+					m_MainMenuUI.visible = visible;
+					if (!temporary)
+						SendVisibilityPulse();
+				}
+			}
+		}
+
+		public bool GetVisible()
```

Comment:
Hm. I don't really like having two ways of setting visibility. I'm fine with renaming GetVisible to a get-only property, but I don't think it is a good idea to have a set_visible as well as SetVisibilty with an extra parameter. I agree that it does break convention, but I think that's less important than DRY in this case. Also, in case you didn't catch the change, this used to be as it was, and I had added a SendVisibilityPulse method to IMainMenu, which was called directly in OnMainMenuActivatorSelected.

@AndrewTHEManeri, thoughts?